include/glow/Partitioner/Partitioner.h

```diff
@@ -29,12 +29,21 @@ using ComputeTimeMapTy = std::unordered_map<Node *, float>;
 using NodesSetTy = std::set<Node *>;
 using PartitionCostMapTy = llvm::DenseMap<Function *, GraphMemInfo>;
 
-/// Helper structure for building a partition. Records 1) a mapping of nodes in
+/// Helper structure for building a partition. Records mapping of nodes in
 /// the original function to destination partitions, along with a list of the
-/// newly-created functions; 2) a mapping of newly-created functions aalong with
-/// a set of nodes sets.
+/// newly-created functions;
 using NodeToFunctionMapTy = llvm::DenseMap<Node *, Function *>;
-using FunctionToNodesMapTy = llvm::DenseMap<Function *, NodesSetTy>;
+
+/// A mapping of newly-created functions along with a set of nodes sets. The
+/// overloaded compare function to make sure the map is sorted by the key's
+/// name(i.e. the function's name) which makes the optimization sequence is
+/// consistent for each run.
+struct ptr_less {
```

Comment:
First, the name `ptr_less` does not follow our usual CamelCase naming convention for types.

And then, it actually compares function names. So, may be it should be called `FunctionNameComparator`?